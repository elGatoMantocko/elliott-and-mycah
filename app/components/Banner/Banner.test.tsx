import { render } from '@testing-library/react';
import React from 'react';
import { ParallaxProvider } from 'react-scroll-parallax';

import { Banner } from '.';

it('should render a <Banner />', () =>
  expect(
    render(
      <ParallaxProvider>
        <Banner image={{ src: 'test-file', min: 'test-file-small' }} height={200} />
      </ParallaxProvider>,
    ).container,
  ).toMatchSnapshot());

it('should render a <Banner>{child}</Banner>', () =>
  expect(
    render(
      <ParallaxProvider>
        <Banner image={{ src: 'test-image', min: 'test-image-small' }} height="50vh">
          <p>Hello world</p>
        </Banner>
      </ParallaxProvider>,
    ).container,
  ).toMatchSnapshot());
