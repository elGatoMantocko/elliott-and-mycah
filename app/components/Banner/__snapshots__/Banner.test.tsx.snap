// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a <Banner /> 1`] = `
<div>
  <div
    class="MuiBox-root css-1gnsnpc"
  >
    <div
      style="position: relative; overflow: hidden; width: 100%; height: 100%; background-color: rgb(204, 204, 204);"
    >
      <div
        data-testid="layer-0"
        style="background-image: url(test-file); background-position: center; background-size: cover; position: absolute; top: 0px; left: 0px; right: 0px; bottom: 0px; will-change: transform;"
      />
    </div>
  </div>
</div>
`;

exports[`should render a <Banner>{child}</Banner> 1`] = `
<div>
  <div
    class="MuiBox-root css-si6xzh"
  >
    <div
      style="position: relative; overflow: hidden; width: 100%; height: 100%; background-color: rgb(204, 204, 204);"
    >
      <div
        data-testid="layer-0"
        style="background-image: url(test-image); background-position: center; background-size: cover; position: absolute; top: 0px; left: 0px; right: 0px; bottom: 0px; will-change: transform;"
      />
      <div
        class="MuiBox-root css-ald1q4"
      >
        <p>
          Hello world
        </p>
      </div>
    </div>
  </div>
</div>
`;
